### PR TITLE
Upgrade librdkafka to 1.5.2 and prepare release 0.11.7

### PR DESCRIPTION
**Pull request by mfelsche**
# Pull request

## Description

Upgrade the dependency on librdkafka to 1.5.2

## Related

* Related Issues: https://github.com/edenhill/librdkafka/issues/2933

## Checklist

* [x] The RFC, if required, has been submitted and approved
* [x] Any user-facing impact of the changes is reflected in docs.tremor.rs
* [x] The code is tested
* [x] Use of unsafe code is reasoned about in a comment
* [x] Update CHANGELOG.md appropriately, recording any changes, bug fixes, or other observable changes in behavior
* [x] The performance impact of the change is measured (see below)

### CODE REVIEW

File: src/source/kafka.rs [hunk 1]
```diff
@@ -152,8 +152,8 @@ impl rentals::MessageStream {
     // ALLOW: https://github.com/tremor-rs/tremor-runtime/issues/1023
     #[allow(mutable_transmutes, clippy::mut_from_ref)]
     unsafe fn mut_suffix(
-        &self,
-    ) -> &mut stream_consumer::MessageStream<'static, LoggingConsumerContext, SmolRuntime> {
+        &mut self,
+    ) -> &mut stream_consumer::MessageStream<'_, LoggingConsumerContext, SmolRuntime> {
```
Comment (Licenser) [inline src/source/kafka.rs:156]: :sob:

**Review by Licenser — APPROVED** [review]
:rocket: :ship: it

File: src/source/kafka.rs (hunk 1 above)
Comment (mfelsche) [reply src/source/kafka.rs:156]: pls don't cry. This change is for the better!

Comment (Licenser) [reply src/source/kafka.rs:156]: I know, but I'll be crying and sobbing in the corner until we finally can get rid of it, each transmute is a tear for me

Comment (mfelsche) [reply src/source/kafka.rs:156]: Each unsafe transmute kills a little kitten when executed.

Comment (Licenser) [reply src/source/kafka.rs:156]: :sob:

**Pull request merged**